intro-HPC/ch_web_portal.tex

```diff
@@ -0,0 +1,217 @@
+\chapter{Using the HPC-UGent web portal}
+\label{ch:web_portal}
+
+\href{https://openondemand.org/}{\strong{Open OnDemand}} (or \texttt{OoD} for short)
```

Comment:
from ohio docs: 
```
We provide a number of web portals for our user community.

OSC OnDemand is our "one stop shop" for access to our High Performance Computing resources. With OnDemand, you can upload and download files, create, edit, submit, and monitor jobs, run GUI applications, and connect via SSH, all via a web broswer, with no client software to install and configure. Full documentation is available. 

```

to get inspired :wink:

Reply:
I got inspired.

Reply:
I would only mention OOD somewhere that the portal is based on it, but keep the terminology on "webportal". this is not relevant in the explanation.

Reply:
eg ` The webportal provides a one stop shop ...`

Reply:
fixed in 119b9af, Open OnDemand is now only mention twice: to mention the web portal is based on it, and when linking to OoD docs.